password.py

```diff
@@ -1,25 +1,60 @@
 #import
 
+#dictionaries for storing accounts,and passwords
 account_dict = {}
-sec_questions_dict = {}
+special_characters = ['!','@','#','$','%','&','*']
+i = 0
```

Comment:
We don't seem to use this anywhere and even if we did its defined a bit too early in the code. Try to define things closer to where they are used, so that its a bit easier to read and understand why they are defined.